packages/ubuntu_desktop_installer/lib/pages/installation_type/installation_type_page.dart

```diff
@@ -127,6 +127,33 @@ class _InstallationTypePageState extends State<InstallationTypePage> {
           onActivated: () async {
             await model.save();
 
+            final telemetry = getService<TelemetryService>();
```

Comment:
The information exchange below is between `telemetry` and `model`. I'd prefer to pass `TelemetryService` from `InstallationTypePage.create()` to `InstallationTypeModel`, and set these telemetry attributes in `InstallationTypeModel.save()`. It would offload this logic from the widget tree and would be easy to test too.

Reply:
Good suggestion. Done.